content/docs-v1/multiformat-output.md

```diff
@@ -79,9 +91,21 @@ Alternatively, you can use [PrinceXML](https://www.princexml.com/) to generate y
 
 To include the PDF file as a download from your online edition:
 
-1. Move the `pagedjs.pdf` (or `prince.pdf`) file into your `content/_assets/downloads/` directory (also rename the file if you’d like)
-2. Confirm that the `resource_link` information in your `content/_data/publication.yaml` file to points to the PDF file you just generated
-3. Run `quire build` again to generate new `_site` files with the PDF included inside
+1. Rename the `pagedjs.pdf` (or `prince.pdf`) file if you would like
+
+2. Move the PDF file into your `content/_assets/downloads/` directory (create the `downloads` directory if it does not already exist)
+
+3. Confirm that the `resource_link` information in your `content/_data/publication.yaml` file to points to the PDF file you just generated
+
+    ```yaml
+    - type: other-format
+      name: PDF
+      media_type: application/pdf
+      link_relation: alternate
+      url: /_assets/downloads/output.pdf
```

Comment:
To parallel the epub example above, shouldn't this be `pagedjs.pdf`? Otherwise, where did `output.pdf` come from?

Reply:
Oh yes, should be `pagedjs.pdf`. Thanks. I'll fix and merge